README.md

```diff
@@ -403,6 +403,10 @@ Destination[] directDestinations = amadeus.airport.directDestinations.get(Params
 // body can be a String version of your JSON or a JsonObject
 FlightAvailability[] flightAvailabilities
   = amadeus.shopping.availability.flightAvailabilities.post(body);
+
+// Travel Restrictions GET
+DiseaseAreaReport diseaseAreaReports 
+  = amadeus.dutyOfCare.diseases.travelRestrictions.get(Params.with("countryCode","US"));
```

Comment:
SInce the endpoint is `/v1/duty-of-care/diseases/covid19-area-report`, it should be dutyOfCare.diseases.travelRestrictions
`amadeus.dutyOfCare.diseases.covid19AreaReport.get(Params.with("countryCode","US"));`

Reply:
Understand! thank you!